tests/lib/openstack_api/test_openstack_hypervisor.py

```diff
@@ -6,119 +7,119 @@ def test_get_state_running():
     Test hypervisor state is running for given variables
     """
     hypervisor = {
-        "hypervisor_uptime": "13:37:30 up 7 days,  4:29, 19 users,  load average: 0.04, 0.04, 0.0",
+        "hypervisor_uptime_days": 7,
         "hypervisor_status": "enabled",
         "hypervisor_state": "up",
         "hypervisor_server_count": 5,
     }
     state = get_hypervisor_state(hypervisor, 60)
-    assert state == "RUNNING"
+    assert state == HypervisorState.RUNNING
 
 
 def test_get_state_pending_maintenance():
     """
     Test hypervisor state is pending maintenace for given variables
     """
     hypervisor = {
-        "hypervisor_uptime": "13:37:30 up 100 days,  4:29, 19 users,  load average: 0.04, 0.04, 0.0",
+        "hypervisor_uptime_days": 100,
         "hypervisor_status": "enabled",
         "hypervisor_state": "up",
         "hypervisor_server_count": 5,
     }
     state = get_hypervisor_state(hypervisor, 60)
-    assert state == "PENDING_MAINTENANCE"
+    assert state == HypervisorState.PENDING_MAINTENANCE
 
 
 def test_get_state_draining():
     """
     Test hypervisor state is draining for given variables
     """
     hypervisor = {
-        "hypervisor_uptime": "13:37:30 up 100 days,  4:29, 19 users,  load average: 0.04, 0.04, 0.0",
+        "hypervisor_uptime_days": 100,
         "hypervisor_status": "disabled",
         "hypervisor_state": "up",
         "hypervisor_server_count": 5,
     }
     state = get_hypervisor_state(hypervisor, 60)
-    assert state == "DRAINING"
+    assert state == HypervisorState.DRAINING
 
 
 def test_get_state_drained():
     """
     Test hypervisor state is drained for given variables
     """
     hypervisor = {
-        "hypervisor_uptime": "13:37:30 up 100 days,  4:29, 19 users,  load average: 0.04, 0.04, 0.0",
+        "hypervisor_uptime_days": 100,
         "hypervisor_status": "disabled",
         "hypervisor_state": "up",
         "hypervisor_server_count": 0,
     }
     state = get_hypervisor_state(hypervisor, 60)
-    assert state == "DRAINED"
+    assert state == HypervisorState.DRAINED
 
 
 def test_get_state_unkown_status():
     """
     Test hypervisor state is unknown when missing status
     """
     hypervisor = {
-        "hypervisor_uptime": "13:37:30 up 100 days,  4:29, 19 users,  load average: 0.04, 0.04, 0.0",
+        "hypervisor_uptime_days": 100,
```

Comment:
Is there an upper and/or lower limit to this or can it be any amount of days?

Reply:
Cool cool, was just wondering if there was a test case for 0 or an upper limit if it existed.